### DISCUSSION TITLE
Stories in main.js outside src

### DISCUSSION THREAD

**Discussion by AndrewOttavianoAbb** [storybookjs/storybook#16323] [Help] [answered]
As the documentation presents it, I should be able to use the `stories` attribute in the config object exported from `main.js` to point to specific locations where I'm keeping my story files. So I tried this by making a new directory (outside of `src`) and trying to find stories in that folder. When I run `npm run storybook` I don't see the story I'm expecting. Am I not configuring something properly or is it not designed to behave this way?

## main.js
```
module.exports = {
  "stories": async (list) => [
    ...list,
    "../src/**/*.stories.mdx",
    "../src/**/**/*.stories.@(js|jsx|ts|tsx)",
    "../hello-chicken/**/*.stories.@(js|jsx|ts|tsx)",
  ],
  "addons": [
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/preset-create-react-app"
  ]
}
```

Project structure:
```
.storybook
  |-main.js
hello-chicken
  |-button.css
  |-Button.jsx
  |-Button.stories.jsx
  |-package.json
src
... (the rest is irrelavent)
```

Note, I have tried various permutations on `"../hello-chicken/**/*.stories.@(js|jsx|ts|tsx)",` such as:
`"../hello-chicken/*.stories.@(js|jsx|ts|tsx)",` and none have made it work. A college had a similar setup to me on SB v. 6.3.8 and it was working for him. He upgraded and it broke. 

I'm running SB v. 6.3.11 on Windows 10 in a React project generated with CRA.

**Comment by shilman** [comment, marked as answer]
Storybook reuses CRA's configuration as closely as possible with `@storybook/preset-create-react-app`. Since CRA doesn't support files outside the `src` directory, neither does Storybook in this configuration.

cc @mrmckeb this comes up quite a bit!

**Reply by AndrewOttavianoAbb** [reply]
Ah makes sense. Thanks!
